cypress/components/dialog/dialog.cy.tsx

```diff
@@ -404,274 +398,60 @@ context("Testing Dialog component", () => {
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog Responsive story", () => {
-      CypressMountWithProviders(<defaultStories.Responsive />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-  });
-
-  describe("Accessibility tests for Dialog component", () => {
-    it("should pass accessibility tests for Dialog default story", () => {
-      CypressMountWithProviders(<stories.DialogComponent />);
-
+      buttonDataComponent().click();
       cy.checkAccessibility();
     });
+  });
 
-    it.each([0, 1, 100, 1000])(
-      "should pass accessibility tests for Dialog component with %s as a height parameter",
-      (height) => {
-        CypressMountWithProviders(
-          <stories.DialogComponent height={`${height}px`} />
-        );
-
-        cy.checkAccessibility();
-      }
-    );
-
-    it.each(specialCharacters)(
-      "should pass accessibility tests for Dialog using %s as a title",
-      (title) => {
-        CypressMountWithProviders(<stories.DialogComponent title={title} />);
-
-        cy.checkAccessibility();
-      }
-    );
-
-    it.each(specialCharacters)(
-      "should pass accessibility tests for Dialog using %s as a subtitle",
-      (subtitle) => {
-        CypressMountWithProviders(
-          <stories.DialogComponent title="Sample dialog" subtitle={subtitle} />
-        );
+  it("should pass accessibility tests for Dialog Editable story", () => {
+    CypressMountWithProviders(<defaultStories.Editable />);
+    openPreviewButton().click();
+    getComponent("icon").should("be.visible");
+    cy.checkAccessibility();
+  });
 
-        cy.checkAccessibility();
-      }
-    );
+  it("should pass accessibility tests for Dialog WithHelp story", () => {
+    CypressMountWithProviders(<defaultStories.WithHelp />);
+    openPreviewButton().click();
+    getComponent("icon").should("be.visible");
+    cy.checkAccessibility();
+  });
 
-    it.each([
-      SIZE.EXTRASMALL,
-      SIZE.SMALL,
-      SIZE.MEDIUMSMALL,
-      SIZE.MEDIUM,
-      SIZE.MEDIUMLARGE,
-      SIZE.LARGE,
-      SIZE.EXTRALARGE,
-    ])(
-      "should pass accessibility tests for Dialog component with %s as a size",
-      (size) => {
-        CypressMountWithProviders(<stories.DialogComponent size={size} />);
+  it("should pass accessibility tests for Dialog DynamicContent story", () => {
+    CypressMountWithProviders(<defaultStories.DynamicContent />);
+    openPreviewButton().click();
+    getComponent("icon").should("be.visible");
+    cy.checkAccessibility();
+  });
 
-        cy.checkAccessibility();
-      }
+  it("should pass accessibility tests for Dialog FocusingADifferentFirstElement story", () => {
```

Comment:
As above, I think this test case should be within the brackets of the other accessibility tests `Describe...`. It's out on its own here.